tests/security/apparmor_profile/usr_sbin_dovecot.pm

```diff
@@ -32,8 +32,11 @@ sub run {
 
     # verify "dovecot" service
     assert_script_run("systemctl stop dovecot.service");
+    sleep 1;
     assert_script_run("systemctl start dovecot.service");
+    sleep 1;
     assert_script_run("systemctl restart dovecot.service");
+    sleep 1;
     assert_script_run("systemctl status --no-pager dovecot.service", sub { m/Active: active (running)./ });
```

Comment:
I wonder if we can do again `wait_still_screen` given that we might be on a serial terminal and i wonder if systemd has a mechanism to notify or send some signal that indicates that x action is now finished.

Reply:
No!

Reply:
There is no reason to replace simple 1 sec wait with complicated function!

Reply:
> There is no reason to replace simple 1 sec wait with complicated function!

sort of, I'm more thinking into how `Utils::Systemd` manages this, and I'm sure that for most cases service changes move fast, so the sleep is ok, but in other cases, these would take longer, but that's a different story